dropwizard-testing/src/main/java/io/dropwizard/testing/common/DropwizardTestResourceConfig.java

```diff
@@ -46,7 +46,21 @@
         register(new JacksonFeature(configuration.mapper));
         register(new HibernateValidationBinder(configuration.validator));
         for (Supplier<?> singleton : configuration.singletons) {
-            register(singleton.get());
+            Object object = singleton.get();
+//            Class<?> clazz = object.getClass();
```

Comment:
Do we want to clean this thing?

Reply:
yep, that's where I was struggling with getting the PersonResource tests to pass & not warn, updated PR now.